docs/ref/databases.txt

```diff
@@ -439,6 +439,24 @@ more details.
 .. _MySQL option file: https://dev.mysql.com/doc/refman/5.6/en/option-files.html
 .. _MySQLdb documentation: http://mysql-python.sourceforge.net/
 
+.. _mysql-sql-mode:
+
+Setting ``sql_mode``
+~~~~~~~~~~~~~~~~~~~~
+
+From MySQL 5.7 onwards, the default value of the ``sql_mode`` option contains
+``STRICT_TRANS_TABLES``. That option is escalating warnings into errors when
+data are truncated upon insertion, so Django highly recommends activating a
+`strict mode`_ for MySQL to prevent data losses (either ``STRICT_TRANS_TABLES``,
+``STRICT_ALL_TABLES``, or ``TRADITIONAL``).
```

Comment:
as below, I wouldn't mention TRADITIONAL, as it just maps on to a selection of modes, including both `STRICT_TRANS_TABLES` and `STRICT_ALL_TABLES`.